python/pyarrow/public-api.pxi

```diff
@@ -130,7 +130,8 @@ cdef object pyarrow_wrap_metadata(
     return result
 
 
-cdef shared_ptr[CKeyValueMetadata] pyarrow_unwrap_metadata(object meta):
+cdef shared_ptr[CKeyValueMetadata] pyarrow_unwrap_metadata(object meta) \
```

Comment:
Does this also introduce a change in the generated C++ header? Do we need to update https://github.com/apache/arrow/blob/master/cpp/src/arrow/python/pyarrow_api.h ?

Reply:
Not really, but updating anyway.